dotCMS/src/main/java/com/dotmarketing/startup/StartupTasksExecutor.java

```diff
@@ -1,343 +1,200 @@
-import com.dotmarketing.util.TaskLocatorUtil;
-
 import java.sql.Connection;
 import java.sql.Date;
-import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
-import java.sql.Statement;
 import java.util.Calendar;
 import java.util.Collections;
 import java.util.Comparator;
 import java.util.List;
 
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.common.reindex.ReindexThread;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.db.LocalTransaction;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.runalways.Task00001LoadSchema;
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.TaskLocatorUtil;
+
+
 public class StartupTasksExecutor {
 
 	
 	private static StartupTasksExecutor executor;
 
-	private String pgLock = "lock table db_version;";
-	private String myLock = "lock table db_version write;";
-	private String oraLock = "LOCK TABLE DB_VERSION IN EXCLUSIVE MODE";
-	private String msLock = "SELECT * FROM db_version WITH (XLOCK)";
-	private String h2Lock = "SELECT * FROM db_version FOR UPDATE";
-
-	private String pgCommit = "commit;";
-	private String myCommit = "unlock tables";
-	private String oraCommit = "COMMIT";
-	private String msCommit = "COMMIT";
-	private String h2Commit = "COMMIT";
 
 	private String pgCreate = "CREATE TABLE db_version (db_version integer NOT NULL, date_update timestamp with time zone NOT NULL, CONSTRAINT db_version_pkey PRIMARY KEY (db_version));";
 	private String myCreate = "CREATE TABLE `db_version` (`db_version` INTEGER UNSIGNED NOT NULL,`date_update` DATETIME NOT NULL, PRIMARY KEY (`db_version`))";
 	private String oraCreate = "CREATE TABLE \"DB_VERSION\" ( \"DB_VERSION\" INTEGER NOT NULL , \"DATE_UPDATE\" TIMESTAMP NOT NULL, PRIMARY KEY (\"DB_VERSION\") )";
 	private String msCreate ="CREATE TABLE db_version (	db_version int NOT NULL , date_update datetime NOT NULL, PRIMARY KEY (db_version) )";
 	private String h2Create = "CREATE TABLE db_version (db_version integer NOT NULL, date_update timestamp NOT NULL, CONSTRAINT db_version_pkey PRIMARY KEY (db_version))";
 	
-	private String pgSelect = "SELECT max(db_version) AS db_version FROM db_version";
-	private String mySelect = "SELECT max(db_version) AS db_version FROM db_version";
-	private String oraSelect = "SELECT max(db_version) AS db_version FROM db_version";
-	private String msSelect = "SELECT max(db_version) AS db_version FROM db_version";
-	private String h2Select = "SELECT max(db_version) AS db_version FROM db_version";
 
-	private String lock;
-	private String commit;
-	private String create;
-	private String select;
 
 
+
+	private final String DOTCMS_FIRST_TIME_START ="DOTCMS_FIRST_TIME_START";
+	private final String create;
+	private final String insert = "INSERT INTO db_version (db_version,date_update) VALUES (?,?)";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)